### ISSUE TITLE
Nav focus colors don't match hover styles

### ISSUE THREAD

**Issue by erinchambers** [CDRH/lewisandclark#254]
When tabbing through the main menu, the link text goes black on `:focus` (insufficient contrast), when it should probably go white on focus like `:hover`.
